source/administration/config-settings.rst

```diff
@@ -3742,6 +3752,18 @@ Specify the color of the SAML login button for white labeling purposes. Use a he
 | This feature's ``config.json`` setting is ``"LoginButtonColor": ""`` with string input.                                       |
 +-------------------------------------------------------------------------------------------------------------------------------+
 
+SAML session lengths
```

Comment:
@tapaswenipathak Is this a setting in config.json? I'm curious what the context is for the session length value

Reply:
You can find information https://docs.mattermost.com/administration/config-settings.html#sessions. There is a ticket from you https://github.com/mattermost/docs/issues/2041.

Reply:
@tapaswenipathak Got it, thanks for the reference! 

For this case,

1. instead of listing the parameters of the SAML session lengths, I think it's sufficient to note which of the [session settings](https://github.com/mattermost/docs/blob/master/source/administration/config-settings.rst#sessions) apply to SAML, and note it as part of the setting itself
2. see if there are any improvements we can make to improve readability.

Let me know if that's helpful or if I can help clarify anything?

Reply:
@tapaswenipathak Thanks! Which of these settings then defines the SAML session length? E.g. if one is set to 14 days, another to 30 days, which will be honoured for SAML?

![image](https://user-images.githubusercontent.com/13119842/58658626-e22df200-82ee-11e9-9bbb-16999a7ee33b.png)

Reply:
I think SAML session length is made up of multiple values? if you could let me know if I can write separate heading defining SAML session length, I will write in a separate line.

Reply:
@tapaswenipathak I did some investigation and looks like the setting `SessionLengthSSOInDays`  controls it for SAML as well.

So we can remove the other two notes about SAML session length, and keep it for the one setting. Here's what I would propose

```
Session length for SSO authentication (days)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
This setting defines the session length for SSO authentication, such as GitLab and SAML.

Set the number of days from the last time a user entered their credentials to the expiry of the user's session. If the authentication method is SAML or GitLab, the user may automatically be logged back in to Mattermost if they are already logged in to SAML or GitLab.

After changing this setting, the setting will take effect after the next time the user enters their credentials.

+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| This feature's ``config.json`` setting is ``"SessionLengthSSOInDays" : 30`` with whole number input.                                                                 |
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```